sample.wdl

```diff
@@ -131,26 +126,38 @@ workflow SampleWorkflow {
                     meanQualityByCycle = false,
                     dockerImages = dockerImages
             }
+
+            call talon.LabelReads as executeLabelReadsTranscriptClean {
```

Comment:
same here, `execute` seems redundant to me.

Reply:
As an aside. `execute` is not the WDL name for performing a task, that is `call`. So if this schema were to make sense, `call` should be used as prefix.

The reason execute looks redundant to me is that *all tasks and workflows* are executed. There is no reason to add this execute statement. It distinguishes nothing.